app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -81,7 +81,7 @@ export const DirectoryColumnPopulationPercent: ColumnComponent = (
   experiment,
 ) => (
   <td data-testid="directory-table-cell">
-    {experiment.populationPercent! || <NotSet />}
+    {Math.round(Number(experiment.populationPercent!)) + "%" || <NotSet />}
```

Comment:
Consider using template literal instead of string concatenation:
```suggestion
    {`${Math.round(Number(experiment.populationPercent!))}%` || <NotSet />}
```

Reply:
that's a good point, but when we start drafting the experiment it shows 0% by default, I think <NotSet> is redundant here

Reply:
Yea, makes sense to me. Probably good to just remove it then.

Reply:
Could also do something like `Number(experiment.populationPercent! || 0)` inside the `Math.round(..)`